src/test/matchers/toMatchText.ts

```diff
@@ -45,7 +37,7 @@ export async function toMatchText(
     throw new Error(`${matcherName} must be called during the test`);
   expectType(receiver, receiverType, matcherName);
 
-  const matcherOptions: MatcherHintOptions = {
```

Comment:
Should we copy this type definition for our own safety?

Reply:
It gets checked down below when we pass it in as an argument.